### PR TITLE
Simple typo changes

### PR DESCRIPTION
**Pull request by techcobweb**
##### SUMMARY
Simple typo changes

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
n/a

##### ADDITIONAL INFORMATION
n/a

### CODE REVIEW

**Comment by xiaoping8385** [comment]
Exception: script returned exit code 1

File: _sources/modules/cics_dfhrmutl.rst.txt [hunk 1]
```diff
@@ -17,10 +17,9 @@ cics_dfhrmutl -- CICS Recovery manager utility program
 Synopsis
 --------
 - You can use the recovery manager utility program to do the following work.
-- Examine the setting of the autostart override record on the global catalog.
-- Set or reset the recovery manager autostart override record on the global catalog.
-- Copy that part of the catalog needed for a cold start to a new global catalog.If a new catalog is built using DFHRMUTL, CICS is able to perform only a cold start or an initial start with the new catalog. The performance of these starts will, however, be better than that of a cold or initial start with a full catalog.
-
+  - Examine the setting of the autostart override record on the global catalog.
```
Comment (techcobweb) [inline _sources/modules/cics_dfhrmutl.rst.txt:20]: Formatting of the list was out. Should be nested under the first bullet ?

**Comment by stewartfrancis** [comment]
Closing as we're not shipping the rmutl module

**Comment by xiaoping8385** [comment]
Exception: hudson.AbortException: Fail: the returned code 404 is not in the accepted range: [[100‥399]]

**Pull request closed without merging**